src/renderer/globals.scss

```diff
@@ -190,3 +234,33 @@ a.is-button {
 .modal-card {
   z-index: 200;
 }
+
+.button .is-rounded .is-primary {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-primary {
+  color: $pitch !important;
```

Comment:
I think the primary color should have `frequency` background, no?

Reply:
this is the color of the text in frequency background
<img width="146" alt="Untitled" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/eada162d-563f-4af0-b245-f26bda7e7fbf">